src/lib/db/query.js

```diff
@@ -7,6 +7,8 @@ const totalCount = 'SELECT count(*)::INTEGER FROM lev_audit';
 const fromDate = 'date_time >= $(from)';
 const toDate = 'date_time < $(to)';
 const searchGroup = 'groups::TEXT ILIKE \'%\' || $(group) || \'%\'';
+const searchWithoutGroup = 'groups::TEXT NOT ILIKE \'%\' || $(withoutGroups) || \'%\'';
+const searchWithoutGroups = 'NOT (groups && $(withoutGroups))';
```

Comment:
I think these should just be part of the standard `sqlBuilder` object in the `WHERE` clause list.

Reply:
Extracted as used in multiple queries.